charts/rekor/templates/server/deployment.yaml

```diff
@@ -57,13 +57,16 @@ spec:
                   name: {{ template "rekor.config" . }}
                   key: {{ .Values.server.config.key }}
           args:
+          - "--trillian_log_server.sharding_config={{ .Values.server.sharding.mountPath }}/{{ .Values.server.sharding.filename }}"
```

Comment:
can you move this into the template helper?

https://github.com/sigstore/helm-charts/blob/main/charts/rekor/templates/_helpers.tpl#L301